### PR TITLE
py-clint: new package at 0.5.1

### PR DESCRIPTION
**Pull request by codeandkey**
depends on `py-args` in #15903

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-clint/package.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class PyClint(PythonPackage):
+    """Python Command-line Application Tools"""
+
+    homepage = "https://github.com/kennethreitz-archive/clint"
+    url      = "https://pypi.io/packages/source/c/clint/clint-0.5.1.tar.gz"
+
+    version('0.5.1', sha256='05224c32b1075563d0b16d0015faaf9da43aa214e4a2140e51f08789e7a4c5aa')
+
+    depends_on('py-setuptools', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-clint/package.py:17]:
```suggestion
    depends_on('py-setuptools', type='build')
```

Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-clint/package.py]: updated!

**Comment by codeandkey** [comment]
Looks like CI failed due to `py-args` not being present yet.

**Comment by adamjstewart** [comment]
Yep, you'll need to rebase now that #15903 has been merged.

**Pull request merged**